src/Pkg.jl

```diff
@@ -271,43 +271,44 @@ const develop = API.develop
 const generate = API.generate
 
 """
-!!! note
-    This feature is experimental.
-
     Pkg.dependencies()::Dict{UUID, PackageInfo}
 
+!!! compat "Experimental"
+    This feature is experimental.
+
 Query the dependecy graph.
 The result is a `Dict` that maps a package UUID to a `PackageInfo` struct representing the dependency (a package).
 
-PackageInfo fields:
+# `PackageInfo` Fields
 
-| `field`        | `Description`                                              |
-|:---------------|:-----------------------------------------------------------|
-| `name`         | The name of the package                                    |
```

Comment:
Why not keep these?

Reply:
It just seems like unnecessary formatting i.e. the "field" title already implies that the names are code.